backend/app/management/commands/syncdb.py

```diff
@@ -1,518 +1,537 @@
 """
```

Comment:
Not sure why this diff is so crazy: I commented out the entire file, and wrote a few paragraphs at the top explaining what it was and why we're keeping it around for a bit longer.